code/modules/jobs/job_types/adventurer/types/combat/rare/assassin.dm

```diff
@@ -68,6 +67,8 @@
 					backr = /obj/item/instrument/guitar
 		if("Beggar") //The sole "town" disguise available.
 			H.job = "Beggar"
+			if(H.mind?.assigned_role)
+				H.mind.assigned_role.title = "Beggar"
```

Comment:
Once again assigned_role is a job which is a singleton, you can't do this

Reply:
i am dumb and forgot, fixed